app/src/main/res/values-sw360dp/dimens.xml

```diff
@@ -12,6 +12,7 @@
     <dimen name="dp_300">300dp</dimen>
     <dimen name="dp_50">50dp</dimen>
     <dimen name="dp_20">20dp</dimen>
+    <dimen name="dp_22">22dp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.dp_22` appears to be unused
 Unused resources make applications larger and slow down builds.